### DISCUSSION TITLE
Localized 4M binaries?

### DISCUSSION THREAD

**Discussion by vibejamin** [arendst/Tasmota#21730] [General]
Hi,

just wondering if there are localized versions (preferably german) of Tasmota for 4M flash available?

Thanks!

**Comment by sfromis** [comment]
No. Localized binaries are only created for the base builds, not the more specialized variants, as visible on the page with the release builds: https://ota.tasmota.com/tasmota/release/

**Reply by vibejamin** [reply]
Thank you for your answer!

**Comment by HofeBY** [comment]
You have to build your own.
https://github.com/HofeBY/Tasmota-Firmware
read Liesmich-zuerst.txt
add your configuration
build, install and its done.
